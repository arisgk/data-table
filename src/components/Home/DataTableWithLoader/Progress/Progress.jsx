import React from 'react';
import CircularProgress from 'material-ui/CircularProgress';

const styles = {
  container: {
    flex: 'auto',
    display: 'flex',
    alignItems: 'center',
    justifyContent: 'center',
  },
};

const Progress = () => (
  <div style={styles.container}>
    <CircularProgress size={30} thickness={3} />
  </div>
);

export default Progress;
