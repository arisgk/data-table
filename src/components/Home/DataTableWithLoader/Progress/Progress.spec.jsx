/* global describe, it, expect */
import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Progress from './Progress';

describe('DataTableWithLoader', () => {
  describe('Progress', () => {
    it('renders correctly', () => {
      const wrapper = shallow(
        <Progress />,
      );

      expect(toJson(wrapper)).toMatchSnapshot();
    });
  });
});
