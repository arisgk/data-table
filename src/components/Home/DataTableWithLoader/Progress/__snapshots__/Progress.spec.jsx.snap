// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DataTableWithLoader Progress renders correctly 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "display": "flex",
      "flex": "auto",
      "justifyContent": "center",
    }
  }
>
  <CircularProgress
    max={100}
    min={0}
    mode="indeterminate"
    size={30}
    thickness={3}
    value={0}
  />
</div>
`;
